gateway/src/integration-test/java/com/hurence/webapiservice/historian/HistorianImportJsonVerticleIT.java

```diff
@@ -210,6 +211,23 @@ void checkAddedTimeseriesChunks(VertxTestContext testContext) {
                                     assertEquals(38,doc1.getLong(RESPONSE_CHUNK_SIZE_BYTES_FIELD));
                                     assertTrue(doc1.containsKey(RESPONSE_CHUNK_VERSION_FIELD));
                                     /*assertEquals(1663577207679746048L,doc1.getLong(RESPONSE_CHUNK_VERSION_FIELD));*/
+                                    JsonObject doc2 = docs.getJsonObject(1);
+                                    assertTrue(doc2.containsKey(RESPONSE_METRIC_NAME_FIELD));
```

Comment:
Here you test the same thing two times. Instead if those fields are present in the first document there is no reason that they are not present in the second one. 
THe problem when testing several time the same things is that when you modify the behaviour you have to modify more code. 
Writing test code is not easy. You must test the maximum number of things but you must take care to not write too much test either because it would cost more when refactoring the behaviour. You can leave this test if you want, this is more an advice for latter test writting.

Reply:
done